relay-server/src/services/project.rs

```diff
@@ -710,7 +725,7 @@ impl Project {
 
         match &mut self.state {
             State::Cached(state) => {
-                let state = state.clone();
+                let state = Arc::clone(state);
```

Comment:
Change due to a suggestion in a previous related PR: https://github.com/getsentry/relay/pull/3508/files